packages/analytics/analytics-chart/sandbox/pages/ChartDemoSimple.vue

```diff
@@ -229,9 +229,9 @@ import {
   SimpleChart,
   TopNTable,
 } from '../../src'
+import { SeededRandom } from '@kong-ui-public/analytics-utilities'
 import type { AnalyticsExploreRecord, DisplayBlob, ExploreResultV4, QueryResponseMeta } from '@kong-ui-public/analytics-utilities'
 import type { AnalyticsChartColors, SimpleChartOptions } from '../../src/types'
-import { SeededRandom } from '../utils/SeedRandom'
```

Comment:
@filipgutica this is my fault, I forgot to update the import in a previous PR 😬  Without this fix, the SimpleChart demo doesn't load.